src/modules/item/sagas.ts

```diff
@@ -625,7 +628,8 @@ export function* itemSaga(legacyBuilder: LegacyBuilderAPI, builder: BuilderClien
     while (true) {
       const result: FetchItemsSuccessAction | FetchCollectionItemsSuccessAction = yield take([
         FETCH_ITEMS_SUCCESS,
-        FETCH_COLLECTION_ITEMS_SUCCESS
+        FETCH_COLLECTION_ITEMS_SUCCESS,
+        FETCH_COLLECTION_THUMBNAILS_SUCCESS
```

Comment:
Fetch item entities when thumbnails are fetched successfully to get the item status correctly

Reply:
Can we trigger this when loading the CollectionsPage instead of doing if on the thumbails success? I'm just thinking that in the /collections page, we're going to fetch the entities (because we show thumbnails) for no reason, right?

Reply:
good point! I'll try to trigger this when loading the CollectionsPage. Thanks for the review 🙌

Reply:
removed logic from here and tracked in this new issue: https://github.com/decentraland/builder/issues/2571